src/components/Select/SelectContainer.jsx

```diff
@@ -0,0 +1,150 @@
+"use client"
+import React, {useState} from 'react'
+import Select from './index'
+
+const newsCategory = [
```

Comment:
```suggestion
export const newsCategory = [
```

자동완성 등 다른 곳에서도 활용할 수 있도록 export 해주실 수 있으신가요?

그리고 카테고리 분류 값이 수정되어서 수정된 것으로 반영해주실 수 있으신가요?

수정된 카테고리 분류는 디스코드 fe-공유에 수정된 엑셀 파일 참고해주세요